gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/language/grammar/TraversalMethodVisitor.java

```diff
@@ -1145,6 +1148,23 @@ public GraphTraversal visitTraversalMethod_property_Object_Object_Object(final G
                 GenericLiteralVisitor.getGenericLiteralList(ctx.genericLiteralList()));
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Traversal visitTraversalMethod_property_Cardinality_Object(
```

Comment:
the other code style item we have is to mark variables `final` when possible.

Reply:
Fixed